### ISSUE TITLE
Experiment Module: Change the name of 'experiment surrounds'

### ISSUE THREAD

**Issue by aislinnpearson** [Rothamsted-Ecoinformatics/farm_rothamsted#256]
There is currently some confusion around the naming of the area that borders an experiment. This issue was raised to attempt to clarify some of that confusion

**Experiment Surrounds:** Historically the commercial surrounds of an experiment have been called 'Odds and Ends'. This made sense when an experiment took up a whole field and so the surrounds really were 'odds and ends' but given that many experiments now have a large area of commercial crop surrounding them, the term 'surrounds' seems more appropriate. Still, an understanding of what 'odds and ends' means is important for interpreting historic records.

**Experiment boundary:** there was then some confusion when specifying the experiment module where we used 'experiment surrounds' to refer to what should have been called the 'experiment boundary'.  This is the total area covered by the experiment, including the paths (uncultivated area between the plots) as well as the uncultivated area surrounding the experiment. See for example the below where the experiment boundary is highlighted in red.

Currently, when uploading an experiment, the experiment boundary defaults to 'experiment surrounds' and I was wondering if we could please change this default to 'experiment boundary'.

![image](https://user-images.githubusercontent.com/16349781/184649425-ab9db68a-62bd-4782-a33e-94410cc2b6b6.png)

**Comment by aislinnpearson** [comment]
I realised today that the experiment boundaries are land assets, so they don't appear in the quick forms. Speaking to the team at North Wyke about this they agreed that experiment boundaries are not plant assets because not all experiments contain plants. For example, some at NW are simply land boundaries used to assess different livestock grazing conditions. For this reason I wonder if we should include them as 'other' but make them a location (which I think would cause them to appear in the locations taxonomy?)

**Comment by paul121** [comment]
> I was wondering if we could please change this default to 'experiment boundary'.

Done!

> For example, some at NW are simply land boundaries used to assess different livestock grazing conditions. For this reason I wonder if we should include them as 'other' but make them a location (which I think would cause them to appear in the locations taxonomy?)

I'm not sure if this was a request to change something.. :D But maybe to clarify, the experiment boundary (previously "surrounds") is a land asset with `land_type = 'other'` and `is_location = TRUE`, so I believe this is already the case.

@aislinnpearson  were you maybe referring to this use case at NW, where they may add in additional land boundaries to assess livestock grazing conditions? If so, that sounds like a sensible way to model this with the plan. You could even break up the hierarchy so that the plot assets have different parents (depending on which paddock they are in), but the grandparent could be the same "Experiment boundary" land asset.

I believe we've talked about this grazing paddock use-case a few times now. It seems that it might make sense to make the paddocks that are "permanent" as land assets that can be referenced outside of experiments too.

**Comment by aislinnpearson** [comment]
> I'm not sure if this was a request to change something.. :D

Nope. Nothing needs doing. Just wanted to make sure I had interpreted it correctly, so I could make sure I documented it correctly. :)

> @aislinnpearson were you maybe referring to this use case at NW, where they may add in additional land boundaries to assess livestock grazing conditions? If so, that sounds like a sensible way to model this with the plan. You could even break up the hierarchy so that the plot assets have different parents (depending on which paddock they are in), but the grandparent could be the same "Experiment boundary" land asset.

We decided with North Wyke in the end to leave it as it is (individual fields) but I love the idea of having an experiment boundary which is bigger than the individual fields. For example, we have one experiment which is split over two fields, and we could actually assign plots to fields. 

Just to make sure I understand that correctly, does this mean the hierarchy would look like this:

Experiment boundary
- field 1
-- plot 1
-- plto 2
-- etc...
- field 2
-- plot 10
-- plot 11
-- etc...

The only thing I am not sure about is how that relates to this issue here https://github.com/Rothamsted-Ecoinformatics/farm_rothamsted/issues/259#issuecomment-1224169409 BUT maybe we have to accept that, for the initial version, these edge cases would need to be edited manually?
